src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -324,9 +320,30 @@ def makeSubscriptions(self, workflow):
 
                 try:
                     # In the message bellow we may want to put the list of datasets too
-                    msg = "Making transfer subscriptions for {}".format(workflow['RequestName'])
-                    self.logger.info(msg)
-                    ddmResultList = self.ddm.makeAggRequests(ddmReqList, aggKey='item')
+                    msg = "Making transfer subscriptions for %s"
+                    self.logger.info(msg, workflow['RequestName'])
+
+                    # NOTE:
+                    #    If ddmReqList is empty it means that no destinationOutputMap
+                    #    has been created for this workflow, either because of
+                    #    all output dataTiers for this workflow have been excluded
+                    #    or because the workflow had no output datasets at all
+                    #    (eg. a Harvesting workflow)
+                    #
+                    # NOTE:
```

Comment:
I think these 2 notes could be dropped. Maybe just write inside of the `else` block that we mark the document as `done` because there is nothing else to be done, just move on. The warning message already explains the rest.

Reply:
done